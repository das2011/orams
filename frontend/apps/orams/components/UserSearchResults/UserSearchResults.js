/* eslint-disable */
import React from 'react'
import AUpageAlert from '@gov.au/page-alerts/lib/js/react.js'
import { uniqueID } from 'shared/utils/helpers'
import styles from './UserSearchResults.scss'

const renderTable = (props, users) => {
  if (users && users.length > 0) {
    return (
      <article role="main">
        <div className={styles.headingRow}>
          <div className="row">
            <div className="col-md-4 col-sm-4">Name</div>
            <div className="col-md-4 col-sm-4">Role</div>
            <div className="col-md-4 col-sm-4">Email</div>
          </div>
        </div>
        {users.map((user, id = uniqueID()) =>
          <div key={id} className={styles.userRow}>
            <div className="row user">
              <div className="col-md-4 col-sm-4 user-name">
                <span className={styles.name}>
                  <a
                    onClick={() => {
                      props.history.push(`/user-profile/${user.id}`)
                    }}
                  >
                    {user.name}
                  </a>
                </span>
              </div>
              <div className="col-md-4 col-sm-4 user-role">
                {user.role}
              </div>
              <div className="col-md-4 col-sm-4 user-email">
                {user.emailAddress}
              </div>
            </div>
          </div>
        )}
      </article>
    )
  }

  return (
    <AUpageAlert as="info">
      <h4>There were no users found for your search</h4>
    </AUpageAlert>
  )
}

const UserSearchResults = props => {
  const { searchResults } = props
  const users = searchResults || {}

  return (
    <div className={styles.container}>
      {renderTable(props, users)}
    </div>
  )
}

export default UserSearchResults
