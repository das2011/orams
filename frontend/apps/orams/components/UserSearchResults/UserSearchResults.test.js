import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import expect from 'expect'
import UserSearchResults from './UserSearchResults'

Enzyme.configure({ adapter: new Adapter() })

describe('UserSearchResults', () => {
  describe('when search results does not exist', () => {
    test('renders no results message', () => {
      const wrapper = shallow(<UserSearchResults />)

      expect(wrapper.find('AUpageAlert')).toBeTruthy()
    })
  })

  describe('when searchResults exists', () => {
    const expectedSuppliers = [
      {
        id: 1,
        name: 'Buyer user',
        role: 'buyer',
        email_address: 'buyer@orams.com'
      },
      {
        id: 2,
        name: 'Seller user',
        role: 'seller',
        email_address: 'seller@orams.com'
      }
    ]

    test('there are 2 supplier rows rendered', () => {
      const wrapper = shallow(<UserSearchResults searchResults={expectedSuppliers} />)
      const displayedSupplierRows = wrapper.find('.user')

      expect(displayedSupplierRows.length).toBe(2)
    })

    test('renders the names correctly', () => {
      const wrapper = shallow(<UserSearchResults searchResults={expectedSuppliers} />)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
