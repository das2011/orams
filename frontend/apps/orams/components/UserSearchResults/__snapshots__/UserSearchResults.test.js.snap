// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserSearchResults when searchResults exists renders the names correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <UserSearchResults
    searchResults={
      Array [
        Object {
          "email_address": "buyer@orams.com",
          "id": 1,
          "name": "Buyer user",
          "role": "buyer",
        },
        Object {
          "email_address": "seller@orams.com",
          "id": 2,
          "name": "Seller user",
          "role": "seller",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <article
        role="main"
      >
        <div
          className="headingRow"
        >
          <div
            className="row"
          >
            <div
              className="col-md-4 col-sm-4"
            >
              Name
            </div>
            <div
              className="col-md-4 col-sm-4"
            >
              Role
            </div>
            <div
              className="col-md-4 col-sm-4"
            >
              Email
            </div>
          </div>
        </div>
        <div
          className="userRow"
        >
          <div
            className="row user"
          >
            <div
              className="col-md-4 col-sm-4 user-name"
            >
              <span
                className="name"
              >
                <a
                  onClick={[Function]}
                >
                  Buyer user
                </a>
              </span>
            </div>
            <div
              className="col-md-4 col-sm-4 user-role"
            >
              buyer
            </div>
            <div
              className="col-md-4 col-sm-4 user-email"
            >
              buyer@orams.com
            </div>
          </div>
        </div>
        <div
          className="userRow"
        >
          <div
            className="row user"
          >
            <div
              className="col-md-4 col-sm-4 user-name"
            >
              <span
                className="name"
              >
                <a
                  onClick={[Function]}
                >
                  Seller user
                </a>
              </span>
            </div>
            <div
              className="col-md-4 col-sm-4 user-role"
            >
              seller
            </div>
            <div
              className="col-md-4 col-sm-4 user-email"
            >
              seller@orams.com
            </div>
          </div>
        </div>
      </article>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="headingRow"
          >
            <div
              className="row"
            >
              <div
                className="col-md-4 col-sm-4"
              >
                Name
              </div>
              <div
                className="col-md-4 col-sm-4"
              >
                Role
              </div>
              <div
                className="col-md-4 col-sm-4"
              >
                Email
              </div>
            </div>
          </div>,
          Array [
            <div
              className="userRow"
            >
              <div
                className="row user"
              >
                <div
                  className="col-md-4 col-sm-4 user-name"
                >
                  <span
                    className="name"
                  >
                    <a
                      onClick={[Function]}
                    >
                      Buyer user
                    </a>
                  </span>
                </div>
                <div
                  className="col-md-4 col-sm-4 user-role"
                >
                  buyer
                </div>
                <div
                  className="col-md-4 col-sm-4 user-email"
                >
                  buyer@orams.com
                </div>
              </div>
            </div>,
            <div
              className="userRow"
            >
              <div
                className="row user"
              >
                <div
                  className="col-md-4 col-sm-4 user-name"
                >
                  <span
                    className="name"
                  >
                    <a
                      onClick={[Function]}
                    >
                      Seller user
                    </a>
                  </span>
                </div>
                <div
                  className="col-md-4 col-sm-4 user-role"
                >
                  seller
                </div>
                <div
                  className="col-md-4 col-sm-4 user-email"
                >
                  seller@orams.com
                </div>
              </div>
            </div>,
          ],
        ],
        "role": "main",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="row"
            >
              <div
                className="col-md-4 col-sm-4"
              >
                Name
              </div>
              <div
                className="col-md-4 col-sm-4"
              >
                Role
              </div>
              <div
                className="col-md-4 col-sm-4"
              >
                Email
              </div>
            </div>,
            "className": "headingRow",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="col-md-4 col-sm-4"
                >
                  Name
                </div>,
                <div
                  className="col-md-4 col-sm-4"
                >
                  Role
                </div>,
                <div
                  className="col-md-4 col-sm-4"
                >
                  Email
                </div>,
              ],
              "className": "row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Name",
                  "className": "col-md-4 col-sm-4",
                },
                "ref": null,
                "rendered": "Name",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Role",
                  "className": "col-md-4 col-sm-4",
                },
                "ref": null,
                "rendered": "Role",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Email",
                  "className": "col-md-4 col-sm-4",
                },
                "ref": null,
                "rendered": "Email",
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="row user"
            >
              <div
                className="col-md-4 col-sm-4 user-name"
              >
                <span
                  className="name"
                >
                  <a
                    onClick={[Function]}
                  >
                    Buyer user
                  </a>
                </span>
              </div>
              <div
                className="col-md-4 col-sm-4 user-role"
              >
                buyer
              </div>
              <div
                className="col-md-4 col-sm-4 user-email"
              >
                buyer@orams.com
              </div>
            </div>,
            "className": "userRow",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="col-md-4 col-sm-4 user-name"
                >
                  <span
                    className="name"
                  >
                    <a
                      onClick={[Function]}
                    >
                      Buyer user
                    </a>
                  </span>
                </div>,
                <div
                  className="col-md-4 col-sm-4 user-role"
                >
                  buyer
                </div>,
                <div
                  className="col-md-4 col-sm-4 user-email"
                >
                  buyer@orams.com
                </div>,
              ],
              "className": "row user",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <span
                    className="name"
                  >
                    <a
                      onClick={[Function]}
                    >
                      Buyer user
                    </a>
                  </span>,
                  "className": "col-md-4 col-sm-4 user-name",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      onClick={[Function]}
                    >
                      Buyer user
                    </a>,
                    "className": "name",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Buyer user",
                      "onClick": [Function],
                    },
                    "ref": null,
                    "rendered": "Buyer user",
                    "type": "a",
                  },
                  "type": "span",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "buyer",
                  "className": "col-md-4 col-sm-4 user-role",
                },
                "ref": null,
                "rendered": "buyer",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "buyer@orams.com",
                  "className": "col-md-4 col-sm-4 user-email",
                },
                "ref": null,
                "rendered": "buyer@orams.com",
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="row user"
            >
              <div
                className="col-md-4 col-sm-4 user-name"
              >
                <span
                  className="name"
                >
                  <a
                    onClick={[Function]}
                  >
                    Seller user
                  </a>
                </span>
              </div>
              <div
                className="col-md-4 col-sm-4 user-role"
              >
                seller
              </div>
              <div
                className="col-md-4 col-sm-4 user-email"
              >
                seller@orams.com
              </div>
            </div>,
            "className": "userRow",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="col-md-4 col-sm-4 user-name"
                >
                  <span
                    className="name"
                  >
                    <a
                      onClick={[Function]}
                    >
                      Seller user
                    </a>
                  </span>
                </div>,
                <div
                  className="col-md-4 col-sm-4 user-role"
                >
                  seller
                </div>,
                <div
                  className="col-md-4 col-sm-4 user-email"
                >
                  seller@orams.com
                </div>,
              ],
              "className": "row user",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <span
                    className="name"
                  >
                    <a
                      onClick={[Function]}
                    >
                      Seller user
                    </a>
                  </span>,
                  "className": "col-md-4 col-sm-4 user-name",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      onClick={[Function]}
                    >
                      Seller user
                    </a>,
                    "className": "name",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Seller user",
                      "onClick": [Function],
                    },
                    "ref": null,
                    "rendered": "Seller user",
                    "type": "a",
                  },
                  "type": "span",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "seller",
                  "className": "col-md-4 col-sm-4 user-role",
                },
                "ref": null,
                "rendered": "seller",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "seller@orams.com",
                  "className": "col-md-4 col-sm-4 user-email",
                },
                "ref": null,
                "rendered": "seller@orams.com",
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
      ],
      "type": "article",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <article
          role="main"
        >
          <div
            className="headingRow"
          >
            <div
              className="row"
            >
              <div
                className="col-md-4 col-sm-4"
              >
                Name
              </div>
              <div
                className="col-md-4 col-sm-4"
              >
                Role
              </div>
              <div
                className="col-md-4 col-sm-4"
              >
                Email
              </div>
            </div>
          </div>
          <div
            className="userRow"
          >
            <div
              className="row user"
            >
              <div
                className="col-md-4 col-sm-4 user-name"
              >
                <span
                  className="name"
                >
                  <a
                    onClick={[Function]}
                  >
                    Buyer user
                  </a>
                </span>
              </div>
              <div
                className="col-md-4 col-sm-4 user-role"
              >
                buyer
              </div>
              <div
                className="col-md-4 col-sm-4 user-email"
              >
                buyer@orams.com
              </div>
            </div>
          </div>
          <div
            className="userRow"
          >
            <div
              className="row user"
            >
              <div
                className="col-md-4 col-sm-4 user-name"
              >
                <span
                  className="name"
                >
                  <a
                    onClick={[Function]}
                  >
                    Seller user
                  </a>
                </span>
              </div>
              <div
                className="col-md-4 col-sm-4 user-role"
              >
                seller
              </div>
              <div
                className="col-md-4 col-sm-4 user-email"
              >
                seller@orams.com
              </div>
            </div>
          </div>
        </article>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="headingRow"
            >
              <div
                className="row"
              >
                <div
                  className="col-md-4 col-sm-4"
                >
                  Name
                </div>
                <div
                  className="col-md-4 col-sm-4"
                >
                  Role
                </div>
                <div
                  className="col-md-4 col-sm-4"
                >
                  Email
                </div>
              </div>
            </div>,
            Array [
              <div
                className="userRow"
              >
                <div
                  className="row user"
                >
                  <div
                    className="col-md-4 col-sm-4 user-name"
                  >
                    <span
                      className="name"
                    >
                      <a
                        onClick={[Function]}
                      >
                        Buyer user
                      </a>
                    </span>
                  </div>
                  <div
                    className="col-md-4 col-sm-4 user-role"
                  >
                    buyer
                  </div>
                  <div
                    className="col-md-4 col-sm-4 user-email"
                  >
                    buyer@orams.com
                  </div>
                </div>
              </div>,
              <div
                className="userRow"
              >
                <div
                  className="row user"
                >
                  <div
                    className="col-md-4 col-sm-4 user-name"
                  >
                    <span
                      className="name"
                    >
                      <a
                        onClick={[Function]}
                      >
                        Seller user
                      </a>
                    </span>
                  </div>
                  <div
                    className="col-md-4 col-sm-4 user-role"
                  >
                    seller
                  </div>
                  <div
                    className="col-md-4 col-sm-4 user-email"
                  >
                    seller@orams.com
                  </div>
                </div>
              </div>,
            ],
          ],
          "role": "main",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="row"
              >
                <div
                  className="col-md-4 col-sm-4"
                >
                  Name
                </div>
                <div
                  className="col-md-4 col-sm-4"
                >
                  Role
                </div>
                <div
                  className="col-md-4 col-sm-4"
                >
                  Email
                </div>
              </div>,
              "className": "headingRow",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="col-md-4 col-sm-4"
                  >
                    Name
                  </div>,
                  <div
                    className="col-md-4 col-sm-4"
                  >
                    Role
                  </div>,
                  <div
                    className="col-md-4 col-sm-4"
                  >
                    Email
                  </div>,
                ],
                "className": "row",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Name",
                    "className": "col-md-4 col-sm-4",
                  },
                  "ref": null,
                  "rendered": "Name",
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Role",
                    "className": "col-md-4 col-sm-4",
                  },
                  "ref": null,
                  "rendered": "Role",
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Email",
                    "className": "col-md-4 col-sm-4",
                  },
                  "ref": null,
                  "rendered": "Email",
                  "type": "div",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "0",
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="row user"
              >
                <div
                  className="col-md-4 col-sm-4 user-name"
                >
                  <span
                    className="name"
                  >
                    <a
                      onClick={[Function]}
                    >
                      Buyer user
                    </a>
                  </span>
                </div>
                <div
                  className="col-md-4 col-sm-4 user-role"
                >
                  buyer
                </div>
                <div
                  className="col-md-4 col-sm-4 user-email"
                >
                  buyer@orams.com
                </div>
              </div>,
              "className": "userRow",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="col-md-4 col-sm-4 user-name"
                  >
                    <span
                      className="name"
                    >
                      <a
                        onClick={[Function]}
                      >
                        Buyer user
                      </a>
                    </span>
                  </div>,
                  <div
                    className="col-md-4 col-sm-4 user-role"
                  >
                    buyer
                  </div>,
                  <div
                    className="col-md-4 col-sm-4 user-email"
                  >
                    buyer@orams.com
                  </div>,
                ],
                "className": "row user",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <span
                      className="name"
                    >
                      <a
                        onClick={[Function]}
                      >
                        Buyer user
                      </a>
                    </span>,
                    "className": "col-md-4 col-sm-4 user-name",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        onClick={[Function]}
                      >
                        Buyer user
                      </a>,
                      "className": "name",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Buyer user",
                        "onClick": [Function],
                      },
                      "ref": null,
                      "rendered": "Buyer user",
                      "type": "a",
                    },
                    "type": "span",
                  },
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "buyer",
                    "className": "col-md-4 col-sm-4 user-role",
                  },
                  "ref": null,
                  "rendered": "buyer",
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "buyer@orams.com",
                    "className": "col-md-4 col-sm-4 user-email",
                  },
                  "ref": null,
                  "rendered": "buyer@orams.com",
                  "type": "div",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "1",
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="row user"
              >
                <div
                  className="col-md-4 col-sm-4 user-name"
                >
                  <span
                    className="name"
                  >
                    <a
                      onClick={[Function]}
                    >
                      Seller user
                    </a>
                  </span>
                </div>
                <div
                  className="col-md-4 col-sm-4 user-role"
                >
                  seller
                </div>
                <div
                  className="col-md-4 col-sm-4 user-email"
                >
                  seller@orams.com
                </div>
              </div>,
              "className": "userRow",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="col-md-4 col-sm-4 user-name"
                  >
                    <span
                      className="name"
                    >
                      <a
                        onClick={[Function]}
                      >
                        Seller user
                      </a>
                    </span>
                  </div>,
                  <div
                    className="col-md-4 col-sm-4 user-role"
                  >
                    seller
                  </div>,
                  <div
                    className="col-md-4 col-sm-4 user-email"
                  >
                    seller@orams.com
                  </div>,
                ],
                "className": "row user",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <span
                      className="name"
                    >
                      <a
                        onClick={[Function]}
                      >
                        Seller user
                      </a>
                    </span>,
                    "className": "col-md-4 col-sm-4 user-name",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        onClick={[Function]}
                      >
                        Seller user
                      </a>,
                      "className": "name",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Seller user",
                        "onClick": [Function],
                      },
                      "ref": null,
                      "rendered": "Seller user",
                      "type": "a",
                    },
                    "type": "span",
                  },
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "seller",
                    "className": "col-md-4 col-sm-4 user-role",
                  },
                  "ref": null,
                  "rendered": "seller",
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "seller@orams.com",
                    "className": "col-md-4 col-sm-4 user-email",
                  },
                  "ref": null,
                  "rendered": "seller@orams.com",
                  "type": "div",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
        ],
        "type": "article",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
